corgi/core/migrations/0098_load_missing_container_errata_builds.py

```diff
@@ -0,0 +1,63 @@
+from collections import defaultdict
+
+from django.db import migrations
+from django.db.models import Count
+
+from corgi.collectors.errata_tool import ErrataTool
+from corgi.tasks.errata_tool import slow_load_errata as current_slow_load_errata
+from corgi.tasks.prod_defs import brew_tag_to_et_prefixes
+from corgi.tasks.tagging import NO_MANIFEST_TAG
+
+
+def load_missing_container_errata_build(apps, schema_editor):
+    ProductStream = apps.get_model("core", "ProductStream")
+    CollectorErrataProductVersion = apps.get_model("collectors", "CollectorErrataProductVersion")
+
+    variants_by_stream = defaultdict(set)
+    # Allow this migration to be re-run in case of timeout or other failure
+    # Only get the streams we haven't processed yet
+    for stream in (
+        ProductStream.objects.annotate(num_components=Count("components"))
+        .filter(num_components__gt=0)
```

Comment:
Well, I have tested it and I know it works. I chose this query because it's the same one we use in the update_manifests task. Probably more worth file to update it there than in this migration.